dev/com.ibm.ws.security.openidconnect.clients.common/resources/com/ibm/ws/security/openidconnect/clients/common/resources/OidcClientMessages.nlsprops

```diff
@@ -304,5 +304,17 @@ NO_RECENT_SESSIONS_WITH_CLAIMS=CWWKS1552E: The [{0}] OpenID Connect client canno
 NO_RECENT_SESSIONS_WITH_CLAIMS.explanation=The OpenID Connect client does not have a recent session with an OpenID Connect provider that matches all of the claims that are specified in the message. The claims that are specified in the message must have values in the logout token that match the same claims of an ID token that was received recently.
 NO_RECENT_SESSIONS_WITH_CLAIMS.useraction=Ensure that claims that are specified in the message match an ID token that was recently obtained from an OpenID Connect provider.
 
+VALUE_NOT_HEXADECIMAL=CWWKS1553E: The provided value is not in hexadecimal format, so it cannot be decoded.
+VALUE_NOT_HEXADECIMAL.explanation=The value for an encrypted token is expected to be encoded in hexadecimal format.
```

Comment:
```suggestion
VALUE_NOT_HEXADECIMAL.explanation=The value for an encrypted token must be encoded in hexadecimal format.
```

Reply:
i've removed this and merged it with `OIDC_SESSION_PARSING_FAILURE `